keymanager/src/client/interface.rs

```diff
@@ -14,6 +17,13 @@ use crate::{
 /// Key manager client interface.
 #[async_trait]
 pub trait KeyManagerClient: Send + Sync {
+    /// Key manager runtime identifier this client is connected to. It may be `None` in case the
+    /// identifier is not known yet (e.g. the client has not yet been initialized).
+    fn runtime_id(&self) -> Option<Namespace>;
+
+    /// Key manager runtime signing key used to sign messages from the key manager.
+    fn rsk(&self) -> Option<PublicKey>;
```

Comment:
```suggestion
    fn runtime_signing_key(&self) -> Option<PublicKey>;
```
Maybe full name is better for public methods. Not sure.